matsim/src/main/java/org/matsim/core/router/NetworkRoutingInclAccessEgressModule.java

```diff
@@ -53,279 +54,301 @@
-			result.add( interactionActivity ) ;
-		} else {
-			// don't add another (interaction) activity
-			// TODO: assuming that this is an interaction activity, e.g. walk - drt interaction - walk
-			// Not clear what we should do if it is not an interaction activity (and how that could happen).
-		}
-
-		Id<Link> endLinkId = toFacility.getLinkId();
-		if ( endLinkId==null ) {
-			endLinkId = startLinkId;
-		}
-
-		if (mode.equals(TransportMode.walk)) {
-			Leg egressLeg = populationFactory.createLeg( TransportMode.non_network_walk ) ;
-			egressLeg.setDepartureTime( now );
-		routeBushwhackingLeg(person, egressLeg, startCoord, toFacility.getCoord(), now, startLinkId, endLinkId, populationFactory, config ) ;
-			result.add( egressLeg ) ;
-		} else {
-			Facility fromFacility = FacilitiesUtils.wrapLink(egressActLink);
-			result.addAll(accessEgressToNetworkRouter.calcRoute(fromFacility, toFacility, now, person));
-		}
-	}
-
-	private static boolean isNotNeedingBushwhackingLeg( Facility toFacility ){
-		if ( toFacility.getCoord() == null ) {
-			// facility does not have a coordinate; we cannot bushwhack
-			return true;
-		}
-		// trip ends on link; no need to bushwhack (this is, in fact, not totally clear: might be link on network of other mode)
-		return toFacility instanceof LinkWrapperFacility;
-	}
-
-	private double addBushwhackingLegFromFacilityToLinkIfNecessary( final Facility fromFacility, final Person person,
-									      final Link accessActLink, double now, final List<PlanElement> result,
-									      final PopulationFactory populationFactory, final String stageActivityType,
-									      Config config ) {
-		if ( isNotNeedingBushwhackingLeg( fromFacility ) ) {
-			return now ;
-		}
-
-		Coord endCoord  = accessActLink.getToNode().getCoord() ;
-		// yyyy think about better solution: this may generate long walks along the link. (e.g. orthogonal projection)
-		Gbl.assertNotNull(endCoord);
-
-		if (mode.equals(TransportMode.walk)) {
-			Leg accessLeg = populationFactory.createLeg( TransportMode.non_network_walk ) ;
-			accessLeg.setDepartureTime( now );
-
-			final Id<Link> startLinkId = fromFacility.getLinkId() ;
-			if ( startLinkId==null ){
-				accessActLink.getId();
-			}
 
-		now += routeBushwhackingLeg(person, accessLeg, fromFacility.getCoord(), endCoord, now, startLinkId, accessActLink.getId(), populationFactory,
-				config ) ;
-			// yyyy might be possible to set the link ids to null. kai & dominik, may'16
-
-			result.add( accessLeg ) ;
-		} else {
-			Facility toFacility = FacilitiesUtils.wrapLink(accessActLink);
-			List<? extends PlanElement> accessTrip = accessEgressToNetworkRouter.calcRoute(fromFacility, toFacility, now, person);
-			for (PlanElement planElement: accessTrip) {
-				now = TripRouter.calcEndOfPlanElement( now, planElement, config ) ;
+    private static final Logger log = Logger.getLogger(NetworkRoutingInclAccessEgressModule.class);
```

Comment:
@jfbischoff this diff is hard to follow. Could you reformat spaces to tabs (for indentation)?

Reply:
whoops, that setting apparently did not survive my IDE update... thanks @michalmac